here-naksha-lib-psql/src/main/java/com/here/naksha/lib/psql/PostgresSession.java

```diff
@@ -322,6 +314,16 @@ private static void addPropertyQuery(@NotNull SQL sql, @NotNull POp propertyOp,
       parameter.add(path.get(path.size() - 1));
       return;
     }
+    if (op == POpType.NULL) {
+      addJsonPath(sql, path, path.size());
+      sql.add(" = 'null'");
```

Comment:
Does this work? I think it is expected to be something like "<field> IS NULL" and not like "<field> = 'null'" ... in my view they two are different. Same comment for Not Null as well. Or did I get it wrong?

Reply:
is null - returns also records without property at all
= 'null' - returns only recoreds with property set to null

Reply:
Correct, that's my understanding as well. So, expectation is former, so checking where property is missing or not missing.
If it is latter (i.e. = 'null'), one can anyways use EQ operator and providing additional NULL operator doesn't have different meaning. Or I still miss something?

Reply:
yes and no
1. Pop.eq requires not null value at the moment - so we would have to change it or you would have to use string "null"
2. Using String "null" with eq is not the same, because '"null"' != '"null"'

so either way we need special implementation for null, only difference for you is whether is more convenient to you to use POp.isNull or POp.eq, in my opinion POp.isNull is better, because in that way you are still able to search for jsons that have string nulls like this:
{ "prop": "null" } 
by using POp.eq

Reply:
Ok understood. Conclusion is for Missing / Not Missing check, we have to use EXISTS or NOT(EXISTS) condition. This addresses my main question.